project/common.scala

```diff
@@ -21,14 +21,14 @@ object Common {
   def dispatchOAuthDep =
     "net.databinder" %% "dispatch-oauth" % dispatchVersion
 
-  def integrationTestDeps(sv: String) = (specs2Dep(sv) :: dispatchDeps) map { _ % "test" }
+  def integrationTestDeps(sv: String) = (specs2Dep(sv) :: Nil) map { _ % "test" }
 
   val settings: Seq[Setting[_]] = Defaults.coreDefaultSettings ++ Seq(
     organization := "net.databinder",
 
-    version := "0.8.4",
+    version := "0.8.4-RC1",
```

Comment:
0.8.4 already released http://repo1.maven.org/maven2/net/databinder/unfiltered-filter_2.11/0.8.4/ but missing tag https://github.com/unfiltered/unfiltered/issues/334
I think next version should not "0.8.4-RC1".